docs/getting_started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started with ComposeDB
+
+## Create a schema
+
+* Write a graphQL compatible composite schema in a text file, such as
+
+example: task.schema
+```
+type Task @model(
+    accountRelation: LIST,
+    description: "Basic task"
+) {
+    assignee: String! @length(min:1, max: 30)
+    completed: Boolean
+    content: String! @length(max:2000)
+}
+
+```
+
+* generate a DID key 
+
+If you do not already have a DID key handy, you will need to create one 
+
+If you have downloaded and installed this repo or the [js-glaze](https://github.com/ceramicstudio/js-glaze) repo you can run
+```
+cd packages/cli
+node bin/run.js did:generate-seed
+```
+save this result for use as the did key in the next step.
+
+* Create schema and output encoded composite
+
+`composedb composite:create [graphql_def_file] -k [didkey] -o [output_file.json]`
```

Comment:
gives error "SyntaxError: Non-base16 character" when this cmd is used "composedb composite:create schema.graphql -k  did:key:z6MkrYP78xpdBEWsba2k4kHzPc3NTqJGuV7MRKdN7EEQNjPu -o composite.json"
![Screenshot 2022-07-26 at 7 44 38 PM](https://user-images.githubusercontent.com/47674286/181035135-8e6783d0-a6af-4327-b027-bfee284d2712.png)